.github/workflows/pr_size.yml

```
@@ -7,7 +7,11 @@ jobs:
     runs-on: ubuntu-latest
     name: Label the PR size
     steps:
+<<<<<<< HEAD
```

actionlint

could not parse as YAML: yaml: line 10: could not find expected ':'

(syntax-check)